setup.py

```diff
@@ -1,39 +1,12 @@
-from setuptools import setup, find_packages
+import sys, os
+from setuptools import setup
+from setuptools.config import read_configuration
+from setuputils import find_version
+thisDir=os.path.dirname(__file__)
 
-from setuputils import find_version, read
-
-
-setup(
-    name='astor',
-    version=find_version('astor/__init__.py'),
-    description='Read/rewrite/write Python ASTs',
-    long_description=read('README.rst'),
-    author='Patrick Maupin',
-    author_email='pmaupin@gmail.com',
-    platforms='Independent',
-    url='https://github.com/berkerpeksag/astor',
-    packages=find_packages(exclude=['tests']),
-    py_modules=['setuputils'],
-    classifiers=[
-        'Development Status :: 5 - Production/Stable',
-        'Environment :: Console',
-        'Intended Audience :: Developers',
-        'License :: OSI Approved :: BSD License',
-        'Operating System :: OS Independent',
-        'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.6',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: Implementation',
-        'Programming Language :: Python :: Implementation :: CPython',
-        'Programming Language :: Python :: Implementation :: PyPy',
-        'Topic :: Software Development :: Code Generators',
-        'Topic :: Software Development :: Compilers',
-    ],
-    keywords='ast, codegen, PEP8',
-)
+cfg = read_configuration(os.path.join(thisDir, 'setup.cfg'))
```

Comment:
I wonder if we should convert this to a `here` (or `path`?) function:

```py
def here(*paths):
    return os.path.join(os.path.dirname(__file__), *paths)
```

So we can turn this line to just

```py
config = read_configuration(here('setup.cfg'))
```

Reply:
It would be overkill and waste of `os.path.dirname` calls.

Reply:
1. os.path.dirname is a pretty lightweight operation
2. setup.py is not called by users since we distribute wheels so "waste of calls" is not a valid argument

Reply:
done